dockstore-integration-testing/src/test/java/io/dockstore/client/cli/DAGWorkflowTestIT.java

```diff
@@ -63,7 +63,7 @@ public void resetDBBetweenTests() throws Exception {
     public final SystemOutRule systemOutRule = new SystemOutRule().enableLog();
 
     private WorkflowsApi setupWebService() throws ApiException {
-        ApiClient webClient = WorkflowIT.getWebClient();
+        ApiClient webClient = WorkflowIT.getWebClient("DockstoreTestUser");
```

Comment:
Ditto, there's an IDE refactoring for this. 
Just watch out that it doesn't scoop up the same string in comments, variable names, and the like.